R/xportr_metadata.R

```diff
@@ -0,0 +1,39 @@
+#' Set variable specifications and domain
+#'
+#' @param .df An R object with columns that can be coerced
+#' @param metacore Either a data.frame that has the names of all possible columns
+#'   and their types, or a `Metacore` object from the `Metacore` package. Required
+#'   column names are dataset, variables, type
+#' @param domain Name of the dataset. Ex ADAE/DM. This will be used to subset
+#'   the metacore object. If none is passed it is assumed to be the name of the
+#'   dataset passed in `.df`.
```

Comment:
Please refer to wiki on guidance for filling out the parameter arugments and change `metacore` to `metadata.

Also, after the title can we state the purpose of this function and how it helps reduce the need to fill out additional arguments in the xportr functions.